platform/ios/src/MGLMapboxEvents.m

```diff
@@ -53,14 +51,29 @@ + (nullable instancetype)sharedInstance {
     return _sharedInstance;
 }
 
+// Readonly Reference to MMEEvents UserDefaults reference
++ (NSUserDefaults*)userDefaults {
```

Comment:
Accessing MME UserDefaults instance provides access to a sharable AppGroup Suite. This means data stored here may also be leveraged in extensions if needed `vs standard defaults`